modules/Movecraft/src/main/java/net/countercraft/movecraft/sign/AntiAircraftDirectorSign.java

```diff
@@ -34,7 +34,7 @@ public final void onSignClick(PlayerInteractEvent event) {
         Craft foundCraft = null;
         CraftManager.getInstance().getCraftsInWorld(block.getWorld());
         for (Craft tcraft : CraftManager.getInstance().getCraftsInWorld(block.getWorld())) {
-            if (MathUtils.locationInHitbox(tcraft.getHitBox(), event.getClickedBlock().getLocation()) &&
+            if (MathUtils.locationInHitBox(tcraft.getHitBox(), event.getClickedBlock().getLocation()) &&
```

Comment:
Whenever you use `locationInHitBox`, you probably actually want to use `locIsNearCraftFast`. This adds a buffer of 3 blocks around the craft such that players onto or next to the craft are considered "in" the craft